Lib/venv/__init__.py

```diff
@@ -250,7 +256,7 @@ def setup_python(self, context):
 
             if sysconfig.is_python_build(True):
                 # copy init.tcl
-                for root, dirs, files in os.walk(context.python_dir):
+                for root, _dirs, files in os.walk(context.python_dir):
```

Comment:
why this change?

Reply:
dirs is not used in the loop, this is a flake recommended way to indicate that. Will revert so I get my feature landed :)